### PR TITLE
docs(i18n): contributing guidelines in polish

### PR DESCRIPTION
**Pull request by mathias3**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Wytyczne dotyczące twojego wkładu w FreeCodeCamp
+
+Cześć. 👋
+
+funkcjonowanie freeCodeCamp.org jest możliwe dzięki tysiącom życzliwych wolontariuszy takich jak Ty. Jesteśmy wdzięczni za Twój wkład i cieszymy się, że możemy Cię powitać na pokładzie.
+
+Ściśle egzekwujemy nasz [„Kodeks postępowania”](https://www.freecodecamp.org/code-of-conduct). Poświęć chwilę, aby go przeczytać. Ma tylko 196 słów.
+
+Szczęsliwego rozwijania kodu open source!
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]: "Szczęśliwego"

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+## Oto kilka zabawnych sposobów, w które możesz pomóc
+
+Możesz przyczynić się do jednego z tych:
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]: "[...] do jednego z tych rzeczy:"

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+Przeczytaj nasz [How to Contribute to Open Source Guide](https://github.com/freeCodeCamp/how-to-contribute-to-open-source).
+
+##### Utknęłam na czymś, co nie jest zawarte w tej dokumentacji. Jak mogę uzyskać pomoc?
+
+Możesz poprosić o pomoc:
+
+- [Kategoria "Współtwórcy" na naszym forum publicznym](https://www.freecodecamp.org/forum/c/contributors)
+- [Nasz publiczny współpracownik - chat room na Gitterze](https://gitter.im/FreeCodeCamp/Contributors)
+
+Jesteśmy podekscytowani, że pomożemy Ci wziąć udział we wszystkich tematach, nad którymi chciałbyś pracować. Zachęcamy do zadawania nam pytań na tematy pokrewne, a my chętnie udzielimy wyjaśnień. Upewnij się, że szukasz swojego zapytania przed wysłaniem nowego. Bądź uprzejmy i cierpliwy. Nasza społeczność wolontariuszy i moderatorów jest zawsze w pobliżu, aby poprowadzić Cię przez Twoje pytania.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]: "Jesteśmy podekscytowani, by pomóc Ci ze wzięciu udziału"

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+Ściśle egzekwujemy nasz [„Kodeks postępowania”](https://www.freecodecamp.org/code-of-conduct). Poświęć chwilę, aby go przeczytać. Ma tylko 196 słów.
+
+Szczęsliwego rozwijania kodu open source!
+
+## Oto kilka zabawnych sposobów, w które możesz pomóc
+
+Możesz przyczynić się do jednego z tych:
+
+ 
+1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#przyczyniaj się do tej otwartej źródłowej bazy danych). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.. Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiać błędy na platformie edukacyjnej.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```

**Review by Qondor — CHANGES_REQUESTED** [review]
Corrected few typos and grammar mistakes, but there may be more of them.

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
+# Wytyczne dotyczące twojego wkładu w FreeCodeCamp
+
+Cześć. 👋
+
+funkcjonowanie freeCodeCamp.org jest możliwe dzięki tysiącom życzliwych wolontariuszy takich jak Ty. Jesteśmy wdzięczni za Twój wkład i cieszymy się, że możemy Cię powitać na pokładzie.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Funkcjonowanie freeCodeCamp.org jest możliwe dzięki tysiącom życzliwych wolontariuszy takich jak Ty. Jesteśmy wdzięczni za Twój wkład i cieszymy się, że możemy powitać Cię na pokładzie.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+Ściśle egzekwujemy nasz [„Kodeks postępowania”](https://www.freecodecamp.org/code-of-conduct). Poświęć chwilę, aby go przeczytać. Ma tylko 196 słów.
+
+Szczęśliwego rozwijania kodu open source!
+
+## Oto kilka zabawnych sposobów, w które możesz pomóc
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
## Oto kilka fajnych sposobów, w które możesz pomóc
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+Możesz przyczynić się do jednego z tych rzeczy:
+
+ 
+1. [Przyczyniaj się do tej otwartej źródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
1. [Rozwijaj tę otwartoźródłowej bazy danych](#https://github.com/freeCodeCamp/freeCodeCamp/). Pomóż nam edytować wyzwania związane z kodowaniem lub naprawiaj błędy na platformie edukacyjnej.
```
This is the proper translation of "open-source" https://sjp.pwn.pl/poradnia/haslo/otwartozrodlowy;10114.html

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+2. Pomoc dla kamperów na naszym [forum publicznym](https://www.freecodecamp.org/forum/). [Odpowiedz na pytania dotyczące kodowania](https://www.freecodecamp.org/forum/?max_posts=1) lub [podaj im informacje zwrotne na temat ich projektów kodowania](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1).
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
2. Pomoc dla kamperów na naszym [publicznym forum](https://www.freecodecamp.org/forum/). [Odpowiedz na pytania dotyczące kodowania](https://www.freecodecamp.org/forum/?max_posts=1) lub [podaj im informacje zwrotne na temat ich projektów kodowania](https://www.freecodecamp.org/forum/c/project-feedback?max_posts=1).
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+3. Pomóż nam dodawać napisy (podpisy zamknięte) do naszych [Filmów na kanale YouTube](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ/videos).
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
3. Pomóż nam dodawać napisy do naszych [filmów na kanale YouTube](https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ/videos).
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+
+## Przyczyniaj się do tej otwartej źródłowej bazy danych.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
## Przyczyniaj się do tej otwartoźródłowej bazy danych.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+Nasza baza danych zawiera tysiące [wyzwań związanych z kodowaniem](https://learn.freecodecamp.org) oraz inne kody źródłowe, które zasilają naszą platformę edukacyjną.
+
+##### Twórz, aktualizuj i naprawiaj błędy w naszych wyzwaniach związanych z kodowaniem.
+
+Wszystkie nasze wyzwania związane z kodowaniem są obsługiwane przez społeczność, dostarczającą specjalistyczną wiedzę od wolontariuszy takich jak Ty.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Wszystkie nasze wyzwania związane z kodowaniem są obsługiwane przez społeczność dostarczającą specjalistyczną wiedzę od wolontariuszy takich jak Ty.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+##### Pomóż nam naprawić błędy w platformie edukacyjnej freeCodeCamp.org
+
+Nasza platforma edukacyjna działa na nowoczesnym stosie JavaScript. Posiada różne komponenty, narzędzia i biblioteki, w tym między innymi Node.js, MongoDB, LoopBack, OAuth 2.0, React, Gatsby, Webpack i wiele innych.
+
+Ogólnie rzecz biorąc,
+
+- Mamy serwer API oparty na Node.js.
+- Zestaw aplikacji klienckich opartych na React.
+- Skrypt, którego używamy do oceny naszych projektów front-end.
+rozwijanie naszego stacku wymaga trochę zrozumienia API, ES6 Syntax, i chęci nauki.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion

Rozwijanie naszego stacku wymaga trochę zrozumienia API, ES6 Syntax, i chęci nauki.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+
+Zasadniczo oczekujemy podstawowej znajomości niektórych z wyżej wymienionych technologii, narzędzi i bibliotek. Biorąc to pod uwagę, nie jesteście zobowiązani do bycia ekspertem w tym zakresie, aby móc wnieść swój wkład.
+
+**Jeśli chcesz pomóc nam udoskonalić naszą bazę danych, możesz albo [skonfigurować freeCodeCamp lokalnie](/docs/how-to-setup-freecamp-locally.md) albo użyć Gitpod, darmowego środowiska online dev.**
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
**Jeśli chcesz pomóc nam udoskonalić naszą bazę danych, możesz albo [skonfigurować freeCodeCamp lokalnie](/docs/how-to-setup-freecamp-locally.md) albo użyć Gitpod, darmowego internetowego środowiska deweloperskiego.**
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+
+[![Otwórz w Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/freeCodeCamp/freeCodeCamp)
+
+(Uruchamia gotowe do kodowania środowisko dev dla freecodecamp w przeglądarce.)
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
(Uruchamia gotowe do kodowania środowisko deweloperskie dla freecodecamp w przeglądarce.)
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+
+## Najczęściej zadawane pytania
+
+### Jak mogę zgłosić błąd, którego nie ma na pokładzie?
+
+Jeśli uważasz, że znalazłeś błąd, przeczytaj najpierw artykuł ["Help I've Found a Bug"](https://forum.freecodecamp.org/t/how-to-report-a-bug/19543) i postępuj zgodnie z jego instrukcjami.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Jeśli uważasz, że znalazłeś błąd, przeczytaj najpierw artykuł ["Pomocy znalazłem błąd"](https://forum.freecodecamp.org/t/how-to-report-a-bug/19543) i postępuj zgodnie z jego instrukcjami.
```

Other links have polish names too, so I suppose this one should have as well?

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+
+Jeśli jesteś pewien, że jest to nowy błąd, stwórz nowy problem z GitHub. Upewnij się, że dołączysz jak najwięcej informacji, abyśmy mogli odtworzyć błąd. Mamy predefiniowany szablon problemu, który pomoże Ci w jego rozwiązaniu.
+
+Proszę zwrócić uwagę, że wszelkie problemy, które szukają pomocy przy kodowaniu w przypadku wyzwania, zostaną zamknięte. Śledzenie spraw jest ściśle związane z zagadnieniami związanymi z bazą danych i dyskusjami. W razie wątpliwości, przed sporządzeniem raportu należy [szukać pomocy na forum](https://www.freecodecamp.org/forum).
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Zwróć uwagę, że wszelkie problemy, które szukają pomocy przy kodowaniu w przypadku wyzwania, zostaną zamknięte. Śledzenie spraw jest ściśle związane z zagadnieniami związanymi z bazą danych i dyskusjami. W razie wątpliwości, przed sporządzeniem raportu należy [szukać pomocy na forum](https://www.freecodecamp.org/forum).
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 17]
```diff
@@ -0,0 +1,107 @@
...
+
+##### Jak mogę zgłosić problem bezpieczeństwa?
+
+Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego, wyślij e-mail na adres `security@freecodecamp.org` i natychmiast się temu przyjrzymy.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego wyślij e-mail na adres `security@freecodecamp.org` i natychmiast się temu przyjrzymy.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 18]
```diff
@@ -0,0 +1,107 @@
...
+
+##### Co oznaczają te różne etykiety, które są oznaczone na tematach?
+
+Nasi moderatorzy społeczności [triage](https://en.wikipedia.org/wiki/Software_bug#Bug_management) zgłaszają problemy i wnioski w oparciu o ich priorytet, dotkliwość i inne czynniki. Możesz [znaleźć kompletny słowniczek ich znaczeń tutaj](https://github.com/freecodecamp/freecodecamp/labels).
+
+Powinieneś przejść przez [**`help wanted`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`first timers welcome`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Powinieneś przejść przez [**`potrzebna pomoc`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`nowicjusze mile widziani`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 19]
```diff
@@ -0,0 +1,107 @@
...
+
+Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego, wyślij e-mail na adres `security@freecodecamp.org` i natychmiast się temu przyjrzymy.
+
+##### Co oznaczają te różne etykiety, które są oznaczone na tematach?
+
+Nasi moderatorzy społeczności [triage](https://en.wikipedia.org/wiki/Software_bug#Bug_management) zgłaszają problemy i wnioski w oparciu o ich priorytet, dotkliwość i inne czynniki. Możesz [znaleźć kompletny słowniczek ich znaczeń tutaj](https://github.com/freecodecamp/freecodecamp/labels).
+
+Powinieneś przejść przez [**`help wanted`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`first timers welcome`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 
+
+Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem pracy nad nimi. Jeśli te kwestie nie są jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem nad nimi pracy. Jeśli w kwestiach tych nie jest jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 20]
```diff
@@ -0,0 +1,107 @@
...
+
+#### Znalazłem literówkę, czy powinienem zgłosić problem, zanim złożę prośbę o ciągnięcie?
+
+W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Kwestie są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Problemy są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md (hunk 17 above)
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Proszę nie tworzyć problemów z GitHub w sprawach bezpieczeństwa. Zamiast tego wyślij e-mail na adres `security@freecodecamp.org` a natychmiast się temu przyjrzymy.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md (hunk 18 above)
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Powinieneś przejść przez [**`potrzebna pomoc`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) lub [**`nowicjusze mile widziani`**](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aopen+is%3Aissue+label%3A%22first+timers+welcome%22) w celu szybkiego przeglądu tego, nad czym możesz pracować. 
```

File: docs/i18n-languages/polish/CONTRIBUTING.md (hunk 19 above)
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Są one do pobrania i nie musisz ubiegać się o pozwolenie przed rozpoczęciem nad nimi pracy. Jeśli w kwestiach tych nie jest jasne, co należy zrobić, nie krępuj się zadawać pytań w komentarzach.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md (hunk 20 above)
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
W przypadku literówek i innych zmian w słownictwie, możesz bezpośrednio otwierać żądania ciągnięcia bez wcześniejszego tworzenia problemu. Problemy są bardziej przydatne do omówienia większych problemów związanych z kodem lub aspektami strukturalnymi programu nauczania.
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 21]
```diff
@@ -0,0 +1,107 @@
...
+
+##### Jak mogę dostać problem przypisany do mnie?
+
+Zazwyczaj nie przydziela się problemów nikomu innemu niż długoletni współpracownicy, aby uniknąć ambitnego nie pojawienia się na zajęciach. Zamiast tego postępujemy zgodnie z poniższą polityką, aby być sprawiedliwym dla wszystkich:
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Zazwyczaj nie przydziela się problemów nikomu innemu niż długoletnim współpracownikom, aby uniknąć ambitnego nie dostarczenia wyników. Zamiast tego postępujemy zgodnie z poniższą polityką, aby być sprawiedliwym dla wszystkich:
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 22]
```diff
@@ -0,0 +1,107 @@
...
+   - Czy upewniłeś się, że wszystkie testy przeszły pomyślnie i potwierdziłeś, że wszystko działa lokalnie?
+3. W końcu popieramy prośby o pociągnięcie, które są zgodne z naszymi zaleceniami.
+ - Podążał pan za listą kontrolną? 
+ - Czy podałeś nazwę swojej listy kontrolnej w sposób znaczący?
+
+Nie potrzebujesz żadnego zezwolenia dla spraw, które są oznaczone jako "pomoc poszukiwana" lub "pierwsze powitanie czasowe", jak wyjaśniono wcześniej. Postępuj zgodnie z wytycznymi i otwórz prośbę o pomoc.
+
+##### Jestem nowy w GitHubie i Open Source, od czego powinienem zacząć?
+
+Przeczytaj nasz [How to Contribute to Open Source Guide](https://github.com/freeCodeCamp/how-to-contribute-to-open-source).
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
Przeczytaj nasz [Poradnik jak wnieść wkład do Open Source](https://github.com/freeCodeCamp/how-to-contribute-to-open-source).
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 23]
```diff
@@ -0,0 +1,107 @@
...
+
+##### Utknęłam na czymś, co nie jest zawarte w tej dokumentacji. Jak mogę uzyskać pomoc?
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
##### Utknąłem na czymś, co nie jest zawarte w tej dokumentacji. Jak mogę uzyskać pomoc?
```

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 24]
```diff
@@ -0,0 +1,107 @@
...
+
+Możesz poprosić o pomoc:
+
+- [Kategoria "Współtwórcy" na naszym forum publicznym](https://www.freecodecamp.org/forum/c/contributors)
+- [Nasz publiczny współpracownik - chat room na Gitterze](https://gitter.im/FreeCodeCamp/Contributors)
```
Comment (Qondor) [inline docs/i18n-languages/polish/CONTRIBUTING.md]:
```suggestion
- [Nasz publiczny chat room dla współpracowników na Gitterze](https://gitter.im/FreeCodeCamp/Contributors)
```

**Review by Qondor — CHANGES_REQUESTED** [review]
I have corrected grammar and syntax errors. Seem okay now.

File: docs/i18n-languages/polish/CONTRIBUTING.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (mathias3) [reply docs/i18n-languages/polish/CONTRIBUTING.md]: wymagana poprawka do tej poprawki- masz tu literówkę

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 25]
```diff
@@ -0,0 +1,107 @@
...
+1. Preferowane jest połączenie pierwszego wniosku o wyciągnięcie ręki w każdej sprawie.
+2. W przypadku wielokrotnych próśb o ten sam problem, dajemy pierwszeństwo jakości kodu w próśb o wyciągnięcie. 
+   - Czy dołączyłeś testy? 
+   - Czy złapałeś wszystkie przypadki?
+   - Czy upewniłeś się, że wszystkie testy przeszły pomyślnie i potwierdziłeś, że wszystko działa lokalnie?
+3. W końcu popieramy prośby o pociągnięcie, które są zgodne z naszymi zaleceniami.
+ - Podążał pan za listą kontrolną? 
+ - Czy podałeś nazwę swojej listy kontrolnej w sposób znaczący?
+
+Nie potrzebujesz żadnego zezwolenia dla spraw, które są oznaczone jako "pomoc poszukiwana" lub "pierwsze powitanie czasowe", jak wyjaśniono wcześniej. Postępuj zgodnie z wytycznymi i otwórz prośbę o pomoc.
```
[the remark this answers is not in the document]
Comment (mathias3) [reply docs/i18n-languages/polish/CONTRIBUTING.md]: literówka "potrzeban pomoc"

File: docs/i18n-languages/polish/CONTRIBUTING.md [hunk 26]
```diff
@@ -0,0 +1,107 @@
...
+##### Utknęłam na czymś, co nie jest zawarte w tej dokumentacji. Jak mogę uzyskać pomoc?
+
+Możesz poprosić o pomoc:
+
+- [Kategoria "Współtwórcy" na naszym forum publicznym](https://www.freecodecamp.org/forum/c/contributors)
+- [Nasz publiczny współpracownik - chat room na Gitterze](https://gitter.im/FreeCodeCamp/Contributors)
+
+Jesteśmy podekscytowani, by pomóc Ci we wzięciu udziału we wszystkich tematach, nad którymi chciałbyś pracować. Zachęcamy do zadawania nam pytań na tematy pokrewne, a my chętnie udzielimy wyjaśnień. Upewnij się, że szukasz swojego zapytania przed wysłaniem nowego. Bądź uprzejmy i cierpliwy. Nasza społeczność wolontariuszy i moderatorów jest zawsze w pobliżu, aby poprowadzić Cię przez Twoje pytania.
+
+Jeśli masz wątpliwości, możesz skontaktować się z naszym zespołem programistów platformy, aby Ci w tym pomóc:
```
[the remark this answers is not in the document]
Comment (mathias3) [reply docs/i18n-languages/polish/CONTRIBUTING.md]: done

File: docs/i18n-languages/polish/CONTRIBUTING.md (hunk 25 above)
[the remark this answers is not in the document]
Comment (Qondor) [reply docs/i18n-languages/polish/CONTRIBUTING.md]: Great catch, thanks.

File: docs/i18n-languages/polish/CONTRIBUTING.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (Qondor) [reply docs/i18n-languages/polish/CONTRIBUTING.md]: Corrected too.

**Comment by RandellDawson** [comment]
@mathias3 You have an earlier PR (#37401) which creates a CONTRIBUTING.md file.  Please either close this PR and make the same changes to #37401 or close #37401 and add its changes to this PR.  You also have another PR (#37384) which creates a CONTRIBUTING.md file.  You should only have a single PR which makes changes to the CONTRIBUTING.md file.  There can be other files on the same PR, but no multiple PRs for the same file.  Merging these would cause a substantial merge conflict.

Thank you.

**Review by Qondor — APPROVED** [review]

**Comment by mathias3** [comment]
> @mathias3 You have an earlier PR (#37401) which creates a CONTRIBUTING.md file. Please either close this PR and make the same changes to #37401 or close #37401 and add its changes to this PR. You also have another PR (#37384) which creates a CONTRIBUTING.md file. You should only have a single PR which makes changes to the CONTRIBUTING.md file. There can be other files on the same PR, but no multiple PRs for the same file. Merging these would cause a substantial merge conflict.
> 
> Thank you.
Done. I deleted contributing.md from both (#37401) and(#37384)

**Comment by RandellDawson** [comment]
@mathias3 I should have chosen my words more carefully.  You can't just delete the file from the PR, because that would delete the file from the repo.  Instead you need to revert the changes to the file on the PR, so it is as if you never made any changes to it.

**Comment by mathias3** [comment]
> @mathias3 You have an earlier PR (#37401) which creates a CONTRIBUTING.md file. Please either close this PR and make the same changes to #37401 or close #37401 and add its changes to this PR. You also have another PR (#37384) which creates a CONTRIBUTING.md file. You should only have a single PR which makes changes to the CONTRIBUTING.md file. There can be other files on the same PR, but no multiple PRs for the same file. Merging these would cause a substantial merge conflict.
> 
> Thank you.

ok, i closed all others pull request to avoid merge conflict. All files are in this pr and they were checked by Qondor and few others for  spell check

**Comment by RandellDawson** [comment]
@raisedadead Do we have any reviewers for this one?

**Comment by raisedadead** [comment]
I do not believe we have anyone, but I am going ahead and merging this for the hard work that has been put in by @mathias3 :)

**Pull request merged**